Cargo.toml

```diff
@@ -16,9 +16,8 @@ tokio-util = "0.7"
 tower-service = "0.3.2"
 
 [dev-dependencies]
-h3-quinn = { git = "https://github.com/hyperium/h3" }
```

Comment:
I feel like removing this would make the example no longer compile, since it still depends on `h3-quinn`, no?

I suppose I should setup CI here...

Reply:
h3-quinn is also in `[dependencies]`. I guess `[dependencies]` are also available in examples right?

Yep, we should probably get CI in place first. I'll fix the issues on this branch.

Also, I'm not familiar with the Tokio stack at all. You could just be really direct on any mistakes I made in the PRs. That'll be really helpful : )